gulpfile.js

```diff
@@ -178,7 +178,34 @@ function runKarma(configFile, done) {
     });
 }
 
-gulp.task('user-guide', ['make-schema'], function() {
+gulp.task('oss-attribution', function() {
+    const fs = require('fs-extra');
+    const path = require('path');
+    let moved = false;
+    // oss-attribution-generator expects (reasonably) that the node_modules directory contains the packages, 
+    // which isn't the case in our usual dev setup
```

Comment:
Kind of beside the point I guess, but I wouldn't call this reasonable. Node has a sytem for resolving modules, and it's not there just for fun.